### PR TITLE
Testing/eu async tests

### PR DESCRIPTION
**Pull request by ErinUntermeyer**
#### Type of Change Made 
- [ ] bugfix 
- [ ] new feature 
- [ ] breaking change 
- [x] testing 
- [ ] formatting 
- [ ] styling
#### What's this PR do?
- Writes test in Search and ApplicantPreview
- Fixed the styling on the main container and added rules for components rendered using the main tag
#### Where should the reviewer start?
`Search.test.tsx` or `ApplicantPreview.test.tsx`
#### How should this be manually tested?
Run `npm test` in your terminal
#### Tests are passing
- [x] except the one that is skipped
#### Any background context you want to provide?
- I didn't want to go much further in ApplicantPreview because if we change the data for the skills and values it will break the test anyway.
- Based on the test, it's not rendering the correct fraction for skills/values matches.
#### What are the relevant tickets?
Makes more progress on #50

### CODE REVIEW

File: src/components/Search/Search.test.tsx [hunk 1]
```diff
@@ -87,18 +91,44 @@ describe('Search Component', () => {
...
-	// 	const clearButton = screen.getByRole('button', { name: /clear/i })
-	// 	// need to render some fake options
-	// 	// find one of them (getByText???)
-	// 	// click it
-	// 	// see that state updated with it?
-	// 	// when clear button clicked, see that state is empty?
-	// 	const option1 = screen.getByRole('button', { name: /buttonname/i})
-	// 	fireEvent.click(option1)
-	// 	fireEvent.click(clearButton)
-	// 	expect(clearButton).toBeInTheDocument()
```
Comment (amykarnaze) [inline src/components/Search/Search.test.tsx:101]: A fireEvent.change might work here?

**Review by amykarnaze — APPROVED** [review]
Nice job getting a head start on the ApplicantPreview test.

File: src/components/Search/Search.test.tsx (hunk 1 above)
Comment (ErinUntermeyer) [reply src/components/Search/Search.test.tsx:101]: I had moved this test down to line 113 and it is currently passing but open to other suggestions on how to do it!

**Review by GreysonElkins — APPROVED** [review]
This looks great EU!

**Pull request merged**